packaging/desktop/package_darwin.sh

```diff
@@ -138,6 +138,13 @@ get_deps() {(
   curl -J -L -Ss "$installer_url" | tar zx
 )}
 
+check_ci() {(
+  temp="$(mktemp -d)"
+  (cd "$temp" && npm i github-ci-status)
```

Comment:
you can add github-ci-status to desktop/package.json and then you don't need to do all this temp stuff... e.g. replace all this with

```
$dir/node_modules/.bin/ci --required-tests 3
```